pkg/build/admission/config.go

```
@@ -14,8 +14,8 @@ import (
 func ReadPluginConfig(pluginConfig map[string]configapi.AdmissionPluginConfig, name string, config runtime.Object) error {
 
 	configFilePath, err := pluginconfig.GetPluginConfigFile(pluginConfig, name, "")
```

OMG, ain't nothing better than swallowed error 😞  Good catch!